hub/src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
-logging.level.ch.prevo.open=DEBUG
-open.prevo.hub.config.file=classpath:nodes.yaml
\ No newline at end of file
+logging.level.root=WARN
+logging.level.ch.prevo.open=INFO
+logging.pattern.console=%d{yy-MM-dd HH:mm:ss} [%thread] %-5level %logger{15} - %msg%n
```

Comment:
Since spring provides the possibility of having different configurations for different profiles, I'd suggest do use this configuration only for "prod" profile since as a developer I'd like to have more log output.

Reply:
Sure; though I'd normally have the special case for non-prod environments (dev etc). 
Would prefer it if we could change the log output locally. What's the preferred "spring way"? (And what's the best doc resource? :-))

Reply:
https://devops.datenkollektiv.de/banner.txt/index.html